src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ERR.cs

```diff
@@ -109,7 +110,7 @@ internal static void CheckValidOpenSslHandle(IntPtr handle)
 
         private sealed class OpenSslCryptographicException : CryptographicException
         {
-            internal OpenSslCryptographicException(int errorCode, string message)
+            internal OpenSslCryptographicException(int errorCode, string? message)
```

Comment:
And if `ErrErrorStringN` doesn't return null, this no longer needs to be made nullable, right?

Reply:
right, as that was only usage